cpp/include/nvstrings/NVText.h

```diff
@@ -187,6 +187,17 @@ class NVText
      */
     static NVStrings* create_ngrams(NVStrings& strs, unsigned int ngrams, const char* separator );
 
+    /**
+     * @brief Creates ngrams of the tokens in each string.
+     * 
+     * @param strs Tokens to make into ngrams.
+     * @param delimiter String used to determine tokens.
+     * @param ngrams The 'n' in ngrams. Example, use 2 for bigrams, 3 for trigrams, etc.
+     * @param separator String used to join tokens.
+     * @return The tokens as ngrams.
+     */
```

Comment:
The implementation in `ngram.cu` has a good set of comments to walk through the code. Could you add a small example here as well?

Reply:
> The implementation in `ngram.cu` has a good set of comments to walk through the code. Could you add a small example here as well?

Added an example to the comments.